ui/src/app/zones/views/ZonesListHeader/ZonesListHeader.tsx

```diff
@@ -1,9 +1,18 @@
+import { useSelector } from "react-redux";
+
 import ZonesListTitle from "../ZonesListTitle";
 
 import SectionHeader from "app/base/components/SectionHeader";
+import zoneSelectors from "app/store/zone/selectors";
 
 const ZonesListHeader = (): JSX.Element => {
-  return <SectionHeader title={<ZonesListTitle />}></SectionHeader>;
+  const zones = useSelector(zoneSelectors.count);
```

Comment:
I guess you could add the fetch here as well?

At least `KVMListHeader` that we looked at has it.

Reply:
Yeah we just noticed that, we're gonna add it as part of the form issue, since we're working in the header there.

Reply:
Either is fine, but it's handy to fetch data in the component that needs it in case you end up using the component elsewhere that might not have fetched the data.